MAUI/DataForm/Editors.md

```diff
@@ -0,0 +1,961 @@
+---
+layout: post
+title: Editors in Maui DataForm control | Syncfusion
```

Comment:
**.NET MAUI** DataForm(SfDataForm) => use this in title and description

Reply:
title and description added